### PR TITLE
added safari, imessage, warp, and slack tracking

### PR DESCRIPTION
**Pull request by jfcrockett**
Added the ability to track additional app data.

### CODE REVIEW

**Comment by jfcrockett** [comment]
Really not sure if this is something that is desired, but I added them for my own use so figured I would push them back to be accepted if wanted!

File: WakaTime/WakaTime.swift [hunk 1]
```diff
@@ -158,6 +158,7 @@ enum Category: String {
     case building
     case debugging
     case designing
+    case messaging
```
Comment (alanhamlett) [inline WakaTime/WakaTime.swift:161]: This should be renamed to `communicating` which is the category we use for all voice/text/video messaging.

**Review by alanhamlett — CHANGES_REQUESTED** [review]
Looks good, will merge it once the category is renamed. Here's the available categories:
https://wakatime.com/colors/categories

File: WakaTime/WakaTime.swift (hunk 1 above)
Comment (jfcrockett) [reply WakaTime/WakaTime.swift:161]: perfect! Will fix.

**Pull request closed without merging**